templates/takeovers/_fin-serv-whitepaper-takeover.html

```diff
@@ -0,0 +1,32 @@
+<section lang="en" class="js-takeover is-bordered p-takeover--fin-serv-whitepaper">
+  <div class="p-strip--image is-dark is-deep p-takeover__suru">
+    <div class="row u-vertically-center">
+      <div class="col-7">
+        <h1>Transforming financial services with multi-cloud</h1>
+        <p class="u-no-padding--top">
+          Cloud technologies are sparking a revolution within financial services operations.
+        </p>
+        <p class="col-5 u-hide--large u-hide--medium u-align--center"><img src="{{ ASSET_SERVER_URL }}c52bcce2-financial-takeover-infographic.svg" alt="Financial Services" /></p>
+        <p><a itemprop="url" href="" class="p-button--positive">Download the whitepaper now</a></p>
```

Comment:
Need a link

Reply:
yes, that was in my notes... I see that david has supplied the /engage page pr now.

Reply:
Needs UTMs, cf takeover brief

Reply:
@caldav 	added